### ISSUE TITLE
Applying colors in variabels.scss and matching document title

### ISSUE THREAD

**Issue by hello-chloe-seoyeong** [Cornerstone-CICCC/web-1-final-project-team3-final#15]
We don't have enough time to fix all of things or issues.
So it would be best to first merge the whole pages, combine the header, footer, and other elements, and then have one person create a branch to match the page titles, use color variables, and make any other adjustments.

example of document title `<title>Documents</title>`
index.html -> Home | Calra Beauty
about.html -> About | Calra Beauty
portfolio.html -> My Work | Carla Beauty
service.html -> Service | Carla Beauty

and color case, 
I saw that the same colors were declared in `variables.scss` on the  @koya-tech  branch.
To avoid conflicts, I didn't declare them again. Once everything is merged, I'll update those color names to use the variable names.

**Comment by koya-tech** [comment]
> example of document title <title>Documents</title>
> index.html -> Home | Calra Beauty
> about.html -> About | Calra Beauty
> portfolio.html -> My Work | Carla Beauty
> service.html -> Service | Carla Beauty

already done. but variables.scss modification is not completed
